Antidote/CompactControlsView.h

```diff
@@ -0,0 +1,47 @@
+//
+//  CompactControlsView.h
+//  Antidote
+//
+//  Created by Chuong Vu on 8/10/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@class CompactControlsView;
+
+@protocol CompactControlsViewDelegate <NSObject>
+
+- (void)compactControlsMicButtonPressed:(CompactControlsView *)compactControlsView;
+- (void)compactControlsSpeakerButtonPressed:(CompactControlsView *)compactControlsView;
+- (void)compactControlsResumeButtonPressed:(CompactControlsView *)compactControlsView;
+- (void)compactControlsVideoButtonPressed:(CompactControlsView *)compactControlsView;
+- (void)compactControlsEndCallButtonPressed:(CompactControlsView *)compactControlsView;
```

Comment:
Compact and expanded views can have same protocol for their interface and same protocol for delegate. Pseudo code:

```
@protocol CallControlsViewProtocol
@property (assign, nonatomic) BOOL videoButtonSelected; 
...

@protocol CallControlsViewDelegate
- (void)callControlsViewMicButtonPressed:(id<CallControlsViewProtocol>)view;
...

@interface CompactControlsView <CallControlsViewProtocol>

@property (weak, nonatomic) id<CallControlsViewDelegate> delegate;

@end

// same for expand view
```